### DISCUSSION TITLE
Appwrite between 2 dates  Date like firebase

### DISCUSSION THREAD

**Discussion by stephane2018** [appwrite/appwrite#4789] [Q&A] [answered]
HI I'm using appwrite with react js  and i wanted to no if it's possible to filter collection between two dates like firebase does so
something like this : 
  firebase.database().ref().child("Users").orderByChild('regdate')
  .startAt("2019-01-05").endAt("2019-01-10");

**Comment by stnguyen90** [comment, marked as answer]
You should be able to user `Query.greaterThan()` and `Query.lessThan()` for this like:

```javascript
databases.listDocuments(
    '[DATABASE_ID]',
    '[COLLECTION_ID]',
    [
        Query.greaterThan('regdate', '2019-01-05'),
        Query.lessThan('regdate', '2019-01-10')
    ]
);
```


See our [Databases docs](https://appwrite.io/docs/databases#querying-documents) for more info.

**Reply by stephane2018** [reply]
Thanks very much 🙏🏿

**Comment by stephane2018** [comment]
I have another question, I wanted to know if it is possible to get all the documents of a collection. I have searched in the doc for a way to do this but I can't find it. Apparently you can only retrieve data by applying a limit and the default is 25 I think. But I want to retrieve the whole collection at once.
